src/main/app/pay/payClient.ts

```diff
@@ -151,10 +169,41 @@ export class GovPayClient implements PayClient {
     }
   }
 
-  private preparePaymentRequest (caseReference: string, externalId: string, fees: Fee[]): object {
+  async update (user: User, paymentReference: string, caseReference: string, caseNumber: string): Promise<void> {
+    checkDefined(user, 'User is required')
+    checkNotEmpty(paymentReference, 'Payment reference is required')
+    checkNotEmpty(caseReference, 'Case Reference is required')
+    checkNotEmpty(caseNumber, 'Case Number is required')
+    try {
+      await request.patch({
```

Comment:
Why patch? Just curious if there is a specific need or preference? I don't mind as long as it works :)

Reply:
@robertparkinson PAY supports update of existing resource thru PATCH.

Reply:
I am fine if it works too but we used request.put in other cases in our project. We haven't used patch anywhere. Let me know what you think!